services/gateway/shared/moleculer-io/index.js

```diff
@@ -443,7 +443,9 @@ function makeHandler(svc, handlerItem) {
 				});
 				svc.logger.info(`   <= ${chalk.green.bold('Success')} ${action}`);
 
-				return addJsonRpcEnvelope(id, res);
+				const output = addJsonRpcEnvelope(id, res);
+				if (respond !== undefined) respond(output);
+				return output;
```

Comment:
Please double check if this change has any side effects, else let's update it to be as below:
```suggestion
				if (respond !== undefined) {
					respond(output);
				} else {
					return output;
				}
				
```

Reply:
Stress tests fail if we don't return output. Hence keeping it as it was.

Reply:
Won't `respond` be `undefined` when we run the stress tests?

Reply:
My bad, it worked on the second try. Maybe I made some mistake earlier